### PR TITLE
docs: improvements

### PR DESCRIPTION
**Pull request by JigarJoshi**

### CODE REVIEW

File: docs/buckets/public-bucket.md [hunk 1]
```diff
@@ -37,7 +38,7 @@ $ aws s3api --endpoint-url https://fly.storage.tigris.dev put-object --bucket fo
 }
 ```
 
-Now, we can now access this file without any authentication.
+Now, anyone can view file without authentication.
```
Comment (ovaistariq) [inline docs/buckets/public-bucket.md]: Now, anyone can read the file

Comment (JigarJoshi) [reply docs/buckets/public-bucket.md]: updated.

File: docs/buckets/public-bucket.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ contents of the bucket `foo-public-bucket`.
 
 ## Accessing objects in a public bucket
 
-Objects in a public bucket can be accessed without any authentication.
+Objects in a public bucket can be read without any authentication. but only
```
Comment (ovaistariq) [inline docs/buckets/public-bucket.md:29]: Objects in a public bucket can be read without any authentication. However, only those with access to the bucket can write objects.

File: docs/buckets/public-bucket.md (hunk 1 above)
Comment (JigarJoshi) [reply docs/buckets/public-bucket.md]: you mean `read this` or `read the` file?

File: docs/buckets/public-bucket.md (hunk 2 above)
Comment (JigarJoshi) [reply docs/buckets/public-bucket.md:29]: Objects (by default) in a public bucket can be read without any authentication. However, only those with access to the bucket can write objects.

(should we add that by default in here?)

Comment (ovaistariq) [reply docs/buckets/public-bucket.md:29]: Looks good

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**